bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -87,7 +87,7 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       ps.sorted
     }
 
-    private def when[A](b: Boolean)(a: => A) = if (b) Some(a) else None
+    private def when[A](b: Boolean)(a: => A) = b option a
```

Comment:
Maybe ```when``` could be replaced with the Scalaz operator ```option``` directly? It's a detail though.